src/App.js

```diff
@@ -1,14 +1,14 @@
 import { Route, Routes } from 'react-router-dom';
-import Home from './pages/Home';
-import Categories from './pages/CategoriesPage';
-import Navigation from './components/Header';
+import Books from './pages/Books';
+import Categories from './pages/Categories';
```

Comment:
- [ ] The Categories component should dispatch an action when the button is clicked and should display the returned "Under construction" string on the page.